Workflow/Tests/ConcurrencyTests.swift

```diff
@@ -319,7 +289,8 @@ final class ConcurrencyTests: XCTestCase {
         signal1.send(value: 1)
         signal2.send(value: 2)
 
-        XCTAssertEqual(0, host.rendering.value.count)
+//        TODO: What does it mean that this is no longer true?
+//        XCTAssertEqual(0, host.rendering.value.count)
```

Comment:
@bencochran, @AquaGeek : I think this is okay, right? When we do a `signal.send()` we synchronously update the `rendering`, unlike with the `Worker`s. Just wanted to confirm before removing this.

Reply:
The `.observe(on: QueueScheduler.workflowExecution)` when subscribing to the signal producer inside `ChildWorker` before meant that all worker outputs were received asynchronously on the next run loop cycle. This exposes that behavior change. I’m not entirely convinced that was intentional behavior or an unintended side-effect that got codified into this test though.